pyro/contrib/gp/__init__.py

```diff
@@ -13,3 +19,64 @@ def __init__(self, Xu, name="inducing_points"):
 
     def forward(self):
         return self.inducing_points
+
+
+class Parameterized(nn.Module):
+    """
+    Parameterized module.
+
+    This is a base class for other classes in this Gaussian Process module.
+    The method `set_prior(param, prior)` will be used to set priors to parameters.
```

Comment:
Could you move these descriptions into docstrings of methods? I think this will look better in the html docs. When writing new docs I usually `make docs && open docs/build/html/index.html` to see how they look.

Reply:
Ah, thanks for the tip! Previously, I just look at docs of other codes to figure out how things will be rendered.